### DISCUSSION TITLE
ELECROW DIS07050H

### DISCUSSION THREAD

**Discussion by spienz** [micropython/micropython#15904] [ESP32]
Hy, could anyone help me with building a bin for this Device with an actuell Micropython Version?
https://www.elecrow.com/wiki/esp32-display-502727-intelligent-touch-screen-wi-fi26ble-800480-hmi-display.html#platforms-supported
The Support of Elecrow is not able to help. 
Or does anyone knows a better integrated EDP32 Device with an touch display?

**Comment by robert-hh** [comment]
Try the esp32s3 generic firmware. That should provide a REPL prompt. I do not know of a display driver for the board, but maybe it exists at the awesome MicroPython site.

**Comment by bidrohini** [comment]
This thread may help a bit: https://forum.lvgl.io/t/driver-for-ili9488-display/2667/16
